barretenberg/cpp/src/barretenberg/serialize/msgpack_impl/name_value_pair_macro.hpp

```diff
@@ -35,12 +35,22 @@
                       _28,                                                                                             \
                       _29,                                                                                             \
                       _30,                                                                                             \
+                      _31,                                                                                             \
```

Comment:
extending the hack to allow for the GoblinUltraHonk VerificationKey MsgPack fields, which exceeded 30